core/src/main/java/com/kin/ecosystem/core/util/ErrorUtil.java

```diff
@@ -32,6 +32,8 @@
 	private static final String ACCOUNT_IS_NOT_LOGGED_IN = "Account is not logged in, please call Kin.login(...) first.";
 	private static final String ACCOUNT_NOT_FOUND = "Account not found";
 	private static final String ACCOUNT_HAS_NO_WALLET = "Account has no wallet";
+	private static final String MIGRATION_NEEDED = "Migration is needed";
+	private static final String WALLET_WAS_NOT_CREATED_IN_THIS_APP = "This wallet was not created in this app";
```

Comment:
`private static final String LOCAL_WALLET_NOT_FOUND = "The requested wallet address was not found locally";`